.github/workflows/ci.yaml

```diff
@@ -1,7 +1,9 @@
 ---
 
 on:
-  - pull_request
+  pull_request:
+    paths-ignore:
+      - '.github/**'
```

Comment:
I think it's not working because of this. And does it actually make sense? If we change how we execute tests, we want to test those CI changes?

Reply:
yes, no, maybe, i don't know... i thought it is not necessary to run the tests, if the change is not testable 🤔

Reply:
it is testable. We can test if the CI works (and we should test that IMO)

Reply:
okay, i close this one, and let it like it is